src/components/ExternalEligibility/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { IconArrowUpRight } from '@tabler/icons-react'
+import styles from './index.module.scss'
+import classNames from 'classnames'
+
+export function ExternalEligibility({
+  externalLink
+}: {
+  externalLink: string
+}) {
+  return (
+    <div className={styles.root}>
+      Leaderboard
+      <a
+        href={externalLink}
+        target="_blank"
+        rel="noopener noreferrer"
+        className={classNames(styles.link, 'button-sm', 'weight--bold')}
+      >
+        View
```

Comment:
This a tag should wrap a `Button` of `secondary` type. The only reason it doesn't match figma right now is because we haven't updated the design system in hp/ui yet, but there is an issue in progress for that